recipes/taskflow/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.definitions["CPPTASKFLOW_VERSION"] = self.deps_cpp_info["taskflow"].version
```

Comment:
I can use `cmake_find_package` generator instead, and read `CPPTASKFLOW_VERSION` from Findtaskflow.cmake directly, no tricks.

Reply:
there's no reference to `CPPTASKFLOW_VERSION` here or in taskflow, it it really required?